internal/app/machined/pkg/controllers/siderolink/manager.go

```diff
@@ -235,6 +239,32 @@ func (ctrl *ManagerController) Run(ctx context.Context, r controller.Runtime, lo
 			return fmt.Errorf("error creating address spec: %w", err)
 		}
 
+		if ctrl.pd.grpcPeerAddrPort != "" {
+			var ourAddr netip.AddrPort
+
+			ourAddr, err = netip.ParseAddrPort(ctrl.pd.grpcPeerAddrPort)
+			if err != nil {
+				return err
+			}
+
+			if err = safe.WriterModify(ctx, r, siderolink.NewTunnel(),
+				func(tunnel *siderolink.Tunnel) error {
+					tunnel.TypedSpec().NodeAddress = ourAddr
+					tunnel.TypedSpec().APIEndpoint = ctrl.pd.apiEndpont
+					tunnel.TypedSpec().IFaceName = constants.SideroLinkName
```

Comment:
```suggestion
					tunnel.TypedSpec().LinkName = constants.SideroLinkName
```

we use links everywhere, not interfaces

Reply:
Done!